pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -8,7 +8,8 @@
 // Maximum number of frames to be processed in a single TCP packet. That's also the number of tail calls we'll have.
 // NOTE: we may need to revisit this const if we need to capture more connections.
 #define HTTP2_MAX_FRAMES_ITERATIONS 10
-#define HTTP2_MAX_FRAMES_TO_FILTER  500
+/* #define HTTP2_MAX_FRAMES_TO_FILTER  450 */
+#define HTTP2_MAX_FRAMES_TO_FILTER 200
```

Comment:
Note to reviewers: this was changed because of instruction count issues, and the value is way lower than it needs to be so I didn't have to change it every time I added a log line. Once the PR gets close to approval, I shall figure out what the new maximum value is (and maybe introduce a separate one for TLS).